src/_gettsim/sozialversicherung/rente/altersrente/rentenformel.yaml

```
@@ -0,0 +1,395 @@
+---
+zugangsfaktor_veränderung_pro_jahr:
+  name:
+    de: Veränderung des Zugangsfaktors pro Jahr
```

```suggestion
    de: Veränderung des Zugangsfaktors pro Jahr relativ zu ... 
```
Not sure of the precise term to put there :innocent:

---

It's an absolute change, not a relative one. In the end it's just a factor without a scale, so I'm not sure what to put here.

---

But it is relative to an age (NRA?). "Jahr" is too ambiguous, could be a calendar year, ...

---

Ah yes

```suggestion
    de: Veränderung des Zugangsfaktors je Jahr vorzeitigem/verzögerten Renteneintritt relativ zur Regelaltersgrenze
```

We should add in the description that there are many exceptions to this rule.

---

Thanks! Preferred it a bit shorter, but should be fine now.